docs/Installation Guide.md

```diff
@@ -0,0 +1,319 @@
+  config.vm.box_url = "http://54.183.32.240/vagrant/box/pc-web-05192014-i386.box"
+
+Vagrant.configure("2") do |config|
+  config.vm.network "forwarded_port", guest: 80, host: 8080,
+  auto_correct: true
+end
+  config.vm.network "forwarded_port", guest: 8000, host: 8001
+  config.vm.network "private_network", ip: "192.168.33.10"
+  config.vm.network "public_network"
+
+end
+```
+Save this file as **Vagrantfile** (if you don't already have this file) in the top level directory for the project.
+
+To download the VM, run the following command:
+
+    vagrant up
+
+You must wait a few minutes for the VM to be downloaded completely.
+
+## Using Vagrant
+
+The `vagrant up` command also boots the Virtual Machine.
+
+Once the VM download has completed, upon boot, it may ask you to choose an `Available bridged network interface`. The first option wil work in most cases.
+
+You may come across a message that says `default: Warning: Remote connection disconnect. Retrying...` This message means that the VM is still booting up which is why we cannot establish a connection with it. It is normal to wait on this message for a few minutes (~5 minutes in my case) before we are able to get a connection to the VM. You may need to wait a few minutes until you get a message saying `default: Machine booted and ready!`.
+
+File syncing will work properly after you receive this message: `default: Mounting shared folders`. Please wait for this message before proceeding to the next steps.
+
+Once the VM booted up (and you were able to receive the messages specified above), you can ssh onto the VM by running the command:
+
+    vagrant ssh
+
+You will notice that the project is now synced to this VM by changing directory to **/vagrant** in the virtual machine.
+
+    cd /vagrant
+
+When you make any changes to the project locally, these changes are also reflected (synced) to the project files located in **/vagrant**, and vice versa.
+
+## Install Project Dependencies
+
+On the VM, update and upgrade:
+
+    sudo apt-get update
+    sudo apt-get upgrade
+
+Install project dependencies:
+
+    sudo apt-get install python-dev
+    sudo apt-get install python-psycopg2
+    sudo apt-get install libpq-dev
+    sudo apt-get install python-pip
+
+Install all Python dependencies specified in the [requirements.txt](https://github.com/Nerdylicious/app-web-server/blob/master/requirements.txt) file using pip:
+
+    sudo pip install Django==1.6.5
+    sudo pip install Jinja2==2.7.3
+    sudo pip install MarkupSafe==0.23
+    sudo pip install Pillow==2.5.1
+    sudo pip install dj-database-url==0.3.0
+    sudo pip install dj-static==0.0.6
+    sudo pip install django-toolbelt==0.0.1
+    sudo pip install djangorestframework==2.3.14
+    sudo pip install gunicorn==19.1.0
+    sudo pip install psycopg2==2.5.3
+    sudo pip install static3==0.5.1
+    sudo pip install django-rest-swagger==0.3.2
+    sudo pip install PyYAML==3.11
+
+## Setup PostgreSQL
+
```

Comment:
again i  am wondering if all this setup can be automated in a start_up.sh script and that script can be called from the vagrantfile